content/en/docs/languages/java/sdk.md

```diff
@@ -1,14 +1,16 @@
 ---
 title: Manage Telemetry with SDK
-weight: 11
+linkTitle: Manage Telemetry with SDK
+weight: 12
+aliases: [sdk-concepts]
```

Comment:
Why this alias?

Reply:
I don't know how that got there to be honest. 🤔

Reply:
Haha, then let's drop it.